.github/workflows/ci.yaml

```
@@ -13,7 +13,7 @@ jobs:
   build:
     runs-on: [self-hosted, docker, integration]
     outputs:
-      image-tags: ${{ steps.docker_meta.outputs.tags }}
+      image-tags: ${{ fromJSON(steps.docker_meta.outputs.json).labels['org.opencontainers.image.version'] }}
```

```suggestion
      image-tags: ${{ steps.meta.outputs.version }}
```

Have you tried the `version` output? https://github.com/docker/metadata-action#outputs

---

good catch, let me try another test release.